controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/RoutingPolicy.java

```diff
@@ -98,21 +102,14 @@ public String toString() {
                              zone.value());
     }
 
-    /** Returns the alias to use for the given application cluster in zone */
-    private static String aliasOf(ClusterSpec.Id cluster, ApplicationId application, ZoneId zone) {
-        List<String> parts = List.of(ignorePartIfDefault(cluster.value()),
-                                     ignorePartIfDefault(application.instance().value()),
-                                     application.application().value(),
-                                     application.tenant().value() +
-                                     "." + zone.value() + "." + "vespa.oath.cloud"
-        );
-        return parts.stream()
-                    .filter(Predicate.not(String::isBlank))
-                    .collect(Collectors.joining("--"));
+    /** Returns the endpoint of given rotation */
+    public static Endpoint endpointOf(ApplicationId application, RotationName rotation, SystemName system) {
+        return Endpoint.of(application).target(rotation).on(Port.tls()).directRouting().in(system);
     }
 
-    private static String ignorePartIfDefault(String s) {
-        return "default".equalsIgnoreCase(s) ? "" : s;
+    /** Returns the endpoint of given clsuter */
```

Comment:
typoo

Reply:
Already fixed in 4fac9aefba8b590a5b190ae42cdd7ae9a3fc38e9.